### PR TITLE
creaete entity and update application prop for connection db

### PR DESCRIPTION
**Pull request by huyenanh2110**

### CODE REVIEW

File: src/main/java/com/ww/vetetableshoppingbe/entity/Category.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (nguyenbatrung714) [inline src/main/java/com/ww/vetetableshoppingbe/entity/Category.java:22]: xin chào

File: src/main/java/com/ww/vetetableshoppingbe/entity/Category.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @Column(name = "category_id")
+    private Integer id;
```
Comment (nguyenbatrung714) [inline src/main/java/com/ww/vetetableshoppingbe/entity/Category.java:24]: sdcvsd

File: src/main/java/com/ww/vetetableshoppingbe/entity/Category.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.ColumnDefault;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+
```
Comment (nguyenbatrung714) [inline src/main/java/com/ww/vetetableshoppingbe/entity/Category.java:13]: xin chao

File: src/main/java/com/ww/vetetableshoppingbe/entity/Category.java
Comment (thesonofwind99) [inline src/main/java/com/ww/vetetableshoppingbe/entity/Category.java:1]: hé lô

**Review by linhnguyen3041999 — APPROVED** [review]

**Pull request merged**